sw/device/lib/testing/test_framework/ottf_common.ld

```diff
@@ -239,5 +228,69 @@ SECTIONS {
     *(.freertos.heap)
   } > ram_main
 
+  /**
+   * Non-volatile scratch and counter areas for tests.
+   *
+   * These sections are meant to be used by tests to store data that should
+   * persist across resets. Since `opentitantool` assumes that the signed
+   * region of an image extends until the end, these sections are placed after
+   * all other sections and are NOLOAD so that they don't appear in the binary
+   * and therefore don't end up being covered by the signature.
+   *
+   * Note that these sections are placed in `eflash` instead of `ottf_flash` so
```

Comment:
is this still true? I see `> ottf_flash` below?

Reply:
No, thanks for catching this!